src/styles/variables.scss

```diff
@@ -0,0 +1,21 @@
+$calendar-padding: 10px;
+$width: 100px;
+$height: 100px;
+$border: 1px;
+$gap: 1px;
+$day-color: #eee;
+$day-font-size: 30px;
+$day-font-family: arial;
+$day-font-color: black;
+$day-transition-duration: 0.5s;
+
+$weekset:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+$square: #{(($width + $border * 2) * 7) + ($gap * 6)};
```

Comment:
I think square is not the best name for calendar width. Let's make it descriptive